peachjam/templates/peachjam/legislation_detail.html

```diff
@@ -1,5 +1,6 @@
 {% extends 'peachjam/layouts/document_detail.html' %}
 {% load i18n peachjam %}
+{% block sub-title %}<h2 class="h5 text-muted">{{ document.citation }}</h2>{% endblock %}
```

Comment:
Only if `document.title != document.citation` which is often true for judgments. Also guard against no citation.